src/SourceBuild/content/test/tests.proj

```diff
@@ -21,6 +21,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="$(TasksDir)Microsoft.DotNet.UnifiedBuild.Tasks\Microsoft.DotNet.UnifiedBuild.Tasks.csproj" />
```

Comment:
From the entire PR, this is the only change that I don't understand and which looks wrong to me. Why would we need to build a tasks assembly here in tests.proj?

Reply:
This reference isn't necessary anymore. Removed with https://github.com/dotnet/sdk/pull/48327/commits/e87b02f8ea3b7ff7ec672d775b9f9d5784470529